library/src/test/java/kwasm/format/text/LimitsTest.kt

```diff
@@ -25,53 +26,59 @@ import org.junit.runners.JUnit4
 @UseExperimental(ExperimentalUnsignedTypes::class)
 class LimitsTest {
 
+    private val context = ParseContext("TokenizerTest.wasm", 1, 1)
+    private val tokenizer = Tokenizer()
+
     @Test
     fun parseSingleNumber_setMinToCorrectValue() {
         val expectedMin = 123456.toUInt()
         val expectedMax = UInt.MAX_VALUE
-        val limits = Type.Limits("123456")
-        Truth.assertThat(limits.value.min).isEqualTo(expectedMin)
-        Truth.assertThat(limits.value.max).isEqualTo(expectedMax)
+        val tokens = tokenizer.tokenize("$expectedMin", context)
+        val parseResult = tokens.parseLimits(0)
+        Truth.assertThat(parseResult.astNode.min).isEqualTo(expectedMin)
+        Truth.assertThat(parseResult.astNode.max).isEqualTo(expectedMax)
```

Comment:
Nit: prefer to import the `assertThat` functions directly, rather than calling them off of `Truth`

Reply:
Done.